### PR TITLE
Add documentation for RTSP server

### PR DESCRIPTION
**Pull request by wicadmin**
Add documentation for RTSP server

### CODE REVIEW

**Comment by cedricve** [comment]
@wicadmin let me know when I can review ;) thanks a lot!

**Comment by wicadmin** [comment]
Yes, you can review.

File: 73_addons/5_using_v4l2_rtsp_server_to_stream.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+## Using V4L2 and RTSP server
+The following shows how to use V4L2 capture with a RTSP server to stream from the Raspberry Pi camera to Kerberos.io running somewhere else (In my case kerberos.io is running in a docker container on a central server)
```
Comment (cedricve) [inline 73_addons/5_using_v4l2_rtsp_server_to_stream.md]: I would not say "My case", but make it more general.

"from the Raspberry Pi camera to Kerberos.io (for example on another Raspberry Pi or Docker container)

File: 73_addons/5_using_v4l2_rtsp_server_to_stream.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+cd
+git clone https://github.com/mpromonet/v4l2rtspserver.git
+cd v4l2rtspserver/
+cmake .
+make
+sudo make install
+```
+
+## Usage
+If you want to run it now, issue the following command:
```
Comment (cedricve) [inline 73_addons/5_using_v4l2_rtsp_server_to_stream.md]: Iw ould execute "To start the livestream, issue .."

File: 73_addons/5_using_v4l2_rtsp_server_to_stream.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+```
+v4l2rtspserver -F15 -H 972 -W 1296 -P 8555 /dev/video0
+```
+and put the following into kerberos.io configuration page (visuals in the next section):
+```rtsp://<kerberos.io ip>:8555/unicast```
+
+Additonal configuration parameters can be found [here](https://github.com/mpromonet/v4l2rtspserver)
+
+## Automate
+To automate running the RTSP server at boot
```
Comment (cedricve) [inline 73_addons/5_using_v4l2_rtsp_server_to_stream.md]: you forgot a "." at the end of the sentence.

**Comment by wicadmin** [comment]
I believe I have the changes incorporated. Thanks.

**Pull request closed without merging**